contracts/test/utils/MockToken.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract MockToken is ERC20, Ownable {
```

Comment:
is this used in the tests?

Reply:
Nope, this is being used in the `Rollup` test for the value of `stakeToken` in order to initialize a Rollup. Will remove from this PR.